languages/python/sqlalchemy-oso/sqlalchemy_oso/compat.py

```diff
@@ -0,0 +1,25 @@
+"""SQLAlchemy version compatibility tools.
+
+Keep us compatible with multiple SQLAlchemy versions by implementing wrappers
+when needed here.
+"""
+from packaging.version import parse
+
+import sqlalchemy
+
+version = parse(sqlalchemy.__version__)  # type: ignore
+USING_SQLAlchemy_v1_3 = version >= parse("1.3") and version < parse("1.4")
```

Comment:
Since this is a constant it should be all uppercase

Reply:
agreed to not upcase